docs/reference/health/health.asciidoc

```diff
@@ -24,42 +28,217 @@ NOTE: {cloud-only}
 [[health-api-desc]]
 ==== {api-description-title}
 
-The health API returns a the health status of an Elasticsearch cluster. It
+The health API returns the health status of an Elasticsearch cluster. It
 returns a list of components that compose Elasticsearch functionality. Each
 component's health is determined by health indicators associated with the
 component.
 
-Each indicator has a health status of: `green`, `yellow` or `red`. The indicator will
+Each indicator has a health status of: `green`, `unknown`, `yellow` or `red`. The indicator will
 provide an explanation and metadata describing the reason for its current health status.
 
 A component's status is controlled by the worst indicator status. The cluster's status
 is controlled by the worst component status.
 
+[[health-api-path-params]]
+==== {api-path-parms-title}
+
+
+`<component>`::
+    (Optional, string) Limits the information returned to the specific component.
+    A comma-separated list of the following options:
++
+--
+  `cluster_coordination`::
+      All health reporting that pertains to successful, stable, and continued membership
+      in a cluster.
```

Comment:
```suggestion
    All health reporting that pertains to a single, stable elected master which 
coordinates other nodes to ensure they all have a consistent view of the
<<cluster-state, cluster state>>.  If your cluster doesn’t have a stable
master, many of its features won’t work correctly. You must fix the master
node’s instability before addressing these other issues. It will not be
possible to solve any other issues while the master node is unstable.
```

Maybe someone from @elastic/es-distributed  team can help review the description on the cluster coordination? This text will also be used by UI via a ℹ️ tooltip to expand on the component. 

Text is taken from https://www.elastic.co/blog/a-new-era-for-cluster-coordination-in-elasticsearch and https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-discovery-hosts-providers.html